docs/source/coremltools.models.rst

```diff
@@ -0,0 +1,67 @@
+Models
+==========================
+
+MLModel
```

Comment:
I'd suggest alphabetizing the entries in this file. I really liked that entries in other files were alphabetized.

Reply:
Good point, but this order was already established in the [existing version of the API Reference](https://coremltools.readme.io/reference/convertersconvert) and I'm just copying it. We can make this part of the review of the next iteration.